README.md

```diff
@@ -1 +1,25 @@
+# C# - Code Snippets
+This project serves as a guide to help you build an application with FreeClimb. View this tutorial on [FreeClimb.com]
 
+## Setting up your new app within your FreeClimb account
+
+To get started using a FreeClimb account, follow the instructions [here](https://docs.freeclimb.com/docs/getting-started-with-freeclimb).
```

Comment:
Since these are meant to be snippets, lets mention that each file should be taken into consideration as an "example". The developer is meant to take the snippet, copy it into their respective project and potentially massage it to work appropriately. If they were to copy this repo as is right now, it should be expected to fail compilation.

Reply:
Something like this?

"This project serves as a guide to help you build an application with FreeClimb. Each snippet represents a different example of a FreeClimb tutorial."